zork++/src/bin/main.rs

```diff
@@ -10,4 +16,19 @@ fn main() {
     log::warn!("warn");
     log::info!("info");
     log::error!("error");
+
+    create_template_project(&Some("proyect".to_owned()), true, &Some(CppCompiler::CLANG));
```

Comment:
You are passing default useless values. Remove then from the call site (here) or explictly pass a valid reference to the `parser_cli` args